scripts/backend_test_shards.json

```diff
@@ -235,19 +207,16 @@
         "core.domain.question_jobs_one_off_test",
         "core.domain.moderator_services_test",
         "core.domain.exp_jobs_one_off_test",
-        "core.domain.collection_validators_test",
         "core.domain.suggestion_jobs_one_off_test",
         "core.domain.image_validation_services_test",
-        "core.storage.beam_job.gae_models_test",
-        "core.domain.beam_job_validators_test"
+        "core.storage.beam_job.gae_models_test"
     ],
     "11": [
         "core.platform.taskqueue.cloud_taskqueue_services_test",
         "core.platform.cloud_translate.cloud_translate_services_test",
         "core.platform.cloud_translate.dev_mode_cloud_translate_services_test",
         "core.storage.translation.gae_models_test",
         "core.domain.translation_domain_test",
-        "core.domain.translation_validators_test",
```

Comment:
Can some of the tests be moved from Shard 11 to Shards 4, 5, 6, 7 or 10? This is just to balance the load across shards